exchanges/kraken/kraken_futures.go

```diff
@@ -116,7 +116,7 @@ func (k *Kraken) FuturesSendOrder(ctx context.Context, orderType order.Type, sym
 	size, limitPrice, stopPrice float64) (FuturesSendOrderData, error) {
 	var resp FuturesSendOrderData
 
-	if ioc {
+	if ioc && orderType != order.Market {
```

Comment:
If there is a conflict here I think it might be better just to remove the IOC boolean param altogether and force the caller to supply the correct order type. Let me know what you think.